### DISCUSSION TITLE
how to update the server to the new 1.20

### DISCUSSION THREAD

**Discussion by m4xr1sk** [itzg/docker-minecraft-bedrock-server#332] [Q&A]
how can I update my bedrock server?

**Comment by itzg** [comment]
Restart the container. If you left `VERSION` as the default, then it will query for the latest at startup.

**Comment by chris948** [comment]
strange update. It messed up my allowlist configs and I had to manually add users, which it added to whitelist. I thought they were moving away from that.

**Reply by brendonthiede** [reply]
I noticed that I had a conflict where the `whitelist.json` no longer existed, but I still had `white-list=true` set in the `server.properties`. I had to set an environment variable `WHITE_LIST` to `FALSE`. In the end there should no longer be a `whitelist.json` file, `server.properties` should have `white-list=false` (or perhaps non-existent), there should be an `allowlist.json` file and `server.properties` should have `allow-list=true`

**Comment by maddox** [comment]
I can't get the auto update to run at all, it keeps saying it can't resolve the host.

```
+ echo 'Downloading Bedrock server version 1.20.11.01 ...'
+ curl -v -o /tmp/bedrock-server-1.20.11.01.zip -fsSL https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20.11.01.zip
Downloading Bedrock server version 1.20.11.01 ...
* getaddrinfo() thread failed to start
* Could not resolve host: minecraft.azureedge.net
* Closing connection 0
curl: (6) getaddrinfo() thread failed to start
+ echo 'ERROR failed to download from https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20.11.01.zip'
+ echo '      Double check that the given VERSION is valid'
+ exit 2
ERROR failed to download from https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20.11.01.zip
      Double check that the given VERSION is valid
```

**Reply by biebl** [reply]
Exactly the same for me here

**Reply by maddox** [reply]
I was able to resolve this by getting my Docker version up to date (24.x.x) vs the version I had running (19.x.x). This is per [this thread](https://github.com/itzg/docker-minecraft-bedrock-server/discussions/320#discussioncomment-6542588).
